src/components/TodoList/TodoList.css

```diff
@@ -0,0 +1,83 @@
+.list {
+  padding: 0;
+}
+
+.list__item {
+  background-image: linear-gradient(to bottom,
+  rgb(186, 138, 123), rgb(181, 133, 118),
+  rgb(177, 129, 113), rgb(172, 124, 108),
+  rgb(167, 119, 103), rgb(163, 114, 98),
+  rgb(158, 110, 92), rgb(154, 105, 87),
+  rgb(149, 100, 82), rgb(144, 95, 77),
+  rgb(140, 91, 72), rgb(135, 86, 67));
```

Comment:
```suggestion
  background-image: linear-gradient(to bottom,
    rgb(186, 138, 123), rgb(181, 133, 118),
    rgb(177, 129, 113), rgb(172, 124, 108),
    rgb(167, 119, 103), rgb(163, 114, 98),
    rgb(158, 110, 92), rgb(154, 105, 87),
    rgb(149, 100, 82), rgb(144, 95, 77),
    rgb(140, 91, 72), rgb(135, 86, 67)
  );
```

Reply:
lint automatically fixes it and makes it as in my variant